src/Patch/Manager.php

```diff
@@ -51,19 +51,26 @@ public function __construct(
     /**
      * Applies all needed patches.
      *
+     * @param array $qualityPatches
      * @throws ShellException
      * @throws ConfigException
      */
-    public function apply(): void
+    public function apply(array $qualityPatches = []): void
     {
         $this->logger->notice('Applying patches');
 
         $command = 'php ./vendor/bin/ece-patches apply';
 
+        if ($qualityPatches) {
+            $command .= sprintf(" '%s'", implode("' '", $qualityPatches));
```

Comment:
To use this functionality we must ensure that client has the latest `magento/magento-cloud-patches` version.
I think, before next release of `ece-tools` constraints in composer.json should be updated to `"magento/magento-cloud-patches": "^1.0.6"`